axfw.py

```diff
@@ -74,11 +76,13 @@ def axfw_check_file_and_validate_parameters(ax, firmware_file):
     Validates the .axfw file before attempting a download.
 
     Returns:
-        0 : File is valid, compatible with the device and can be loaded
-        5 : .axfw file is invalid. E.g. failed signature check or CRC mismatch
-        6 : The .axfw is intended for a different aXiom device
-        7 : The .axfw firmware is already on the device
-        8 : The firmware variant in the .axfw does not match the device's firmware variant
+         0 : File is valid, compatible with the device and can be loaded
+         5 : .axfw file is invalid. E.g. failed signature check or CRC mismatch
+         6 : The .axfw is intended for a different aXiom device
+         7 : The .axfw firmware is already on the device
+         8 : The firmware variant in the .axfw does not match the device's firmware variant
+        10 : The device is in bootloader mode already, the device ID is correct, but the firmware variant cannot be
```

Comment:
Would it be worth having the same return code across all scripts for 'device is in bootloader mode'?

Reply:
I have added a common set of return codes to all scripts.